### PR TITLE
Enabling read-by-uuid for records with binary uuids

### PR DESCRIPTION
**Pull request by tamas-molnar**
- added switch for binary uuid
- defaults to false so existing instances will not be affected
- only used for read by uuid so far

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -39,6 +39,7 @@ func main() {
 	app := cli.App("restorage", "A RESTful storage API with pluggable backends")
 	port := app.IntOpt("port", 8080, "Port to listen on")
 	idMap := app.StringOpt("id-map", "test1:uuid,test2:id,...", "Mapping of collection name to identifier property name")
+	isBinaryUUID := app.BoolOpt("binary-uuid", false, "Is the configured id in a binary format?")
```
Comment (ghost) [inline app.go]: should be "binary-identity" or similar because it's not necessarily a UUID.

Comment (ghost) [inline app.go]: Also, this option should only apply to "mongo" mode, not "elastic" or any other.

Comment (tamas-molnar) [inline app.go]: updated

**Comment by catalin-ilea** [comment]
+1

**Pull request merged**